docs/setup/independent/high-availability.md

```diff
@@ -443,7 +442,7 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
     - `<etcd0-ip>`, `<etcd1-ip>` and `<etcd2-ip>` with the IP addresses of your three etcd nodes
     - `<podCIDR>` with your Pod CIDR. Please read the [CNI network section](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the docs for more information. Some CNI providers do not require a value to be set.
 
-    **Note:** If you are using Kubernetes 1.9+, you can replace the `apiserver-count: 3` extra argument with `endpoint-reconciler-type=lease`. For more information, see [the documentation](https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler).
+    **Note:** If you are using Kubernetes 1.9+, you can replace the `apiserver-count: 3` extra argument with `endpoint-reconciler-type: lease`. For more information, see [the documentation](https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler).
```

Comment:
Was the switch from `endpoint-reconciler-type=lease` to `endpoint-reconciler-type: lease` intentional? It doesn’t match the documentation you reference at https://kubernetes.io/docs/admin/high-availability/#endpoint-reconciler, which uses the `=` format, and I suspect is correct.

Reply:
@heckj I did not visit the reference, it's the original link.
as in my experiment with v1.9.0
`endpoint-reconciler-type: lease` is the right one.

Reply:
@heckj 
I guess:

> You can switch to the new reconciler by adding the flag --endpoint-reconciler-type=lease

which means the form:
kubeadm --param-1 --param-2

Reply:
please take a look at my [exp](https://github.com/opera443399/ops/blob/master/doc/CLOUD/k8s/k8s%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2%E6%89%8B%E5%86%8C/%E5%86%8D%E6%8E%A2%E4%BD%BF%E7%94%A8kubeadm%E9%83%A8%E7%BD%B2%E9%AB%98%E5%8F%AF%E7%94%A8%E7%9A%84k8s%E9%9B%86%E7%BE%A4-01%E5%BC%95%E8%A8%80.md#%E9%85%8D%E7%BD%AE-k8s-master)

Reply:
This should stay `endpoint-reconciler-type: lease`.  The `--foo=bar` syntax is when the flag is passed on the command line.  In a YAML file it is `foo: bar`.  This section is talking about the kubeadm config yaml file and as such it should be `foo: bar`.